packages/terra-clinical-result/src/ClinicalResult.jsx

```diff
@@ -106,20 +127,29 @@ const ClinicalResult = (props) => {
     resultData,
     hideUnit,
     isTruncated,
+    isUnverified,
+    isModified,
+    hasComment,
     hasResultError,
     hasResultNoData,
+    hideAccessoryDisplays,
     intl,
     ...customProps
   } = props;
 
+  resultData.isUnverified = isUnverified;
```

Comment:
Why are we adding these values to the results object? Also we don't want to directly manipulate the props they've given us. We should create our own